Assets/Scripts/NPC/NPCBehavior.cs

```diff
@@ -271,8 +299,8 @@ public void PlaySound()
 
         public void SetTimeInterval()
         {
-            _timeInterval = UnityEngine.Random.Range(5.0f, 60.0f);
-            // Debug.Log("Set:" + _timeInterval);
+            _timeInterval = Random.Range(5.0f, 60.0f);
```

Comment:
Why not use the `UnityEngine.Random`? Can't checkout the PR as it has compilation error and the only error I got when opening with text editor is on this line.
```
'Random' is an ambiguous reference between 'UnityEngine.Random' and 'System.Random'
```

Reply:
This isn't my code, it was added after I resolved the merge conflict
I guess the author used `UnityEngine.Random` anyway

I repaired the branch now